### PR TITLE
Fix #1531 error with stdout/stderr pipes

### PR DESCRIPTION
**Pull request by slinkydeveloper**
The #1531 logs to the provided writer the stdout/stderr, providing an utility to log everything that comes in a reader to a logf function. This approach is not working, so i removed the new logging utils and i ported the code to log the result of the stdout/stderr pipes.

### CODE REVIEW

**Comment by n3wscott** [comment]
Can you add a description on why these are deleted?

**Comment by slinkydeveloper** [comment]
done

File: test/monitoring/monitoring.go [hunk 1]
```diff
@@ -74,15 +77,31 @@ func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remo
...
 	if err := c.Start(); err != nil {
 		return nil, fmt.Errorf("%s command failed: %w", cmd, err)
 	}
 	return c.Process, nil
 }
+
+func printPipe(logf logging.FormatLogger, reader io.ReadCloser) {
+	defer reader.Close()
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
```
Comment (vagababov) [inline test/monitoring/monitoring.go:104]: if you want to be fancy
```suggestion
	
	for scanner := bufio.NewScanner(reader); scanner.Scan(); {
```

Comment (slinkydeveloper) [reply test/monitoring/monitoring.go:104]: I prefer two lines :)

**Pull request merged**